scripts/install.js

```diff
@@ -102,7 +102,7 @@ function downloadBinary() {
   const platform = os.platform();
   const outputPath = path.resolve(
     __dirname,
-    platform === 'win32' ? '../bin/sentry-cli.exe' : '../sentry-cli'
+    platform === 'win32' ? '../sentry-cli.exe' : '../sentry-cli'
```

Comment:
Let's update it, so the binaries are downloaded to the same directory. It's important for `is_npm_install` check.